torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16287,6 +16302,13 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
             DecorateInfo(unittest.skip("Skipped!"), "TestJit", "test_variant_consistency_jit", dtypes=(torch.float32,),),
         )
     ),
+    OpInfo(
+        "nn.functional.pdist",
+        ref=reference_pdist,
+        sample_inputs_func=sample_inputs_pdist,
+        dtypes=floating_types(),
+        supports_autograd=False,
```

Comment:
`pdist` not supporting autograd is news to me

Reply:
I'll take a closer look

Reply:
Good catch